mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageMapper.java

```diff
@@ -66,6 +67,7 @@
         .build();
     public static final int EXPUNGE_BATCH_SIZE = 100;
     public static final Logger LOGGER = LoggerFactory.getLogger(CassandraMessageMapper.class);
+    public static final int UPDATE_FLAGS_BATCH_SIZE = 20;
```

Comment:
I do believe we can expose these batch size with use of @aduprat  PR (once merged) to allow admin to configure it. I do belive it might depend on hardware you are using...

Yes for grouping.